homeassistant/components/telegram_bot/bot.py

```diff
@@ -786,6 +786,39 @@ async def leave_chat(self, chat_id=None, context=None, **kwargs):
             self.bot.leave_chat, "Error leaving chat", None, chat_id, context=context
         )
 
+    async def set_message_reaction(
+        self,
+        chat_id: int,
+        reaction: list[str],
+        is_big: bool = False,
+        context: Context | None = None,
+        **kwargs,
+    ) -> None:
+        """Set the bot's reaction for a given message."""
+        chat_id = self._get_target_chat_ids(chat_id)[0]
+        message_id, inline_message_id = self._get_msg_ids(kwargs, chat_id)
+        params = self._get_msg_kwargs(kwargs)
+
+        _LOGGER.debug(
+            "Set reaction to message %s in chat ID %s to %s with params: %s",
+            message_id or inline_message_id,
```

Comment:
```suggestion
            message_id,
```
inline_message_id is not used later